src/engine/app.rs

```diff
@@ -67,7 +68,7 @@ impl Application {
         let fixed_step = self.context.lock().unwrap().fixed_step.clone();
         let last_fixed_update = self.context.lock().unwrap().last_fixed_update.clone();
 
-        if SteadyTime::now() - last_fixed_update > fixed_step {
+        if Instant::now() - last_fixed_update > fixed_step {
             self.states.fixed_update(self.context.lock().unwrap().deref_mut());
             self.context.lock().unwrap().last_fixed_update = last_fixed_update + fixed_step;
```

Comment:
Minor tweak: with the standard library's `Duration` type, I think we can actually use the `AddAssign` operator here. I don't have access to a computer at the moment, so I can't check whether this is the case.